packages/tools/src/synchronizers/callbacks/cameraSyncCallback.ts

```diff
@@ -34,9 +34,14 @@ export default function cameraSyncCallback(
   const tViewport = renderingEngine.getViewport(targetViewport.viewportId);
 
   // TODO: only sync in-plane movements if one viewport is a stack viewport
+  if (camera.parallelProjection) {
```

Comment:
Yes, for now that is acceptable - Eric and I discussed it, and we want an options object to be passed into the synchronizer so that exactly what gets synced can be controlled.  
If you sync everything on any viewport that doesn't start with the same initial conditions, it puts the images offscreen typically, so it wasn't working previously except in a very narrow case.  This version is much more robust, although there are still bugs.